app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    caches = {}
+
+    def inner(*args) -> dict:
+        if args in caches.keys():
```

Comment:
you don't need explicitly mention keys here, since checking a dict will anyway check its keys